### PR TITLE
Wiki Reorg: Unify Statemin* pages

### PR DESCRIPTION
**Pull request by filippoweb3**
See [this issue](https://github.com/w3f/polkadot-wiki/issues/4376) and [this google doc](https://docs.google.com/document/d/1Fl8ReBIzcaQEOyF6s03d7MEcgk8aTVEt7xjmzAk_k5I/edit) for more information about the Wiki reorg.

Statemine and Statemint pages included conditional rendering for both Kusama and Polkadot. Just kept one version of the pages, specifying the screenshots are on one chain but the same applies to the other.

This PR also focuses on renaming titles and URLs to Asset Hub.

### CODE REVIEW

File: docs/learn/learn-assets.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 id: learn-assets
-title: Assets
-sidebar_label: Assets
+title: Polkadot Asset Hub
```
Comment (profradha) [inline docs/learn/learn-assets.md]: Can just be Asset Hub. Depending on whether it is on Polkadot Wiki or Kusama Guide, the readers can deduce that we are discussing about Asset Hub on the respective chains.

```suggestion
title: Asset Hub
```

File: docs/learn/learn-assets.md [hunk 2]
```diff
@@ -50,16 +50,20 @@ to learn how to create fungible assets on Statemine. The same procedure applies
 Anyone on the network can create assets on
 {{ polkadot: Statemint, :polkadot }}{{ kusama: Statemine, :kusama }} as long as they can reserve the
```
Comment (profradha) [inline docs/learn/learn-assets.md]:
```suggestion
Asset Hub as long as they can reserve the
```

File: kusama-guide/sidebars.js [hunk 3]
```diff
@@ -53,7 +53,7 @@ module.exports = {
               items: [
                 "learn/learn-treasury",
                 "learn/learn-assets",
-                "general/kusama/kusama-statemine",
```
Comment (profradha) [inline kusama-guide/sidebars.js:56]: Just like this one, you have to implement redirects for other changed slugs as well so the links on third party websites are not broken.

**Review by DrW3RK — CHANGES_REQUESTED** [review]

File: kusama-guide/sidebars.js (hunk 3 above)
Comment (profradha) [reply kusama-guide/sidebars.js:56]: I'm referring to 

                "learn/learn-staking-guides",
                "learn/learn-identity-guides",
                "learn/learn-assets-guides",
                "learn/learn-vault-guides",

**Comment by filippoweb3** [comment]
@DrW3RK I also fixed the broken links in the previous PR about wallets. There were also some major conflicts with the two previous PRs that now are resolved.

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**